lib/restAPIs/reports.js

```diff
@@ -54,7 +39,7 @@ module.exports = class RestTopic extends RestUtil {
     assert(instanceId, 'No instanceId provided');
     assert(requestBody, 'No requestBody provided');
 
-    const requestOpts = buildLimitQueryParams(offset, limit, fields);
+    const requestOpts = buildCommonQuery(fields, offset, limit);
     return this._makeRequest(`v2/${this._getBase()}/${objectId}/instances/${instanceId}`, requestOpts.queryParams, 'PUT', this.getProjectHeader())
```

Comment:
I think this was a typo (my fault), since the 2nd parmaeter for a PUT request should be the request body (like the POST requests). Could you fix it, while you're here?
```
    return this._makeRequest(`v2/${this._getBase()}/${objectId}/instances/${instanceId}`, requestBody, 'PUT', this.getProjectHeader(), requestOpts)
```